jsonrpc/filter_manager.go

```diff
@@ -17,7 +16,7 @@ import (
 )
 
 var (
-	ErrFilterDoesNotExists              = errors.New("filter does not exists")
+	ErrFilterDoesNotExists              = errors.New("filter not found")
```

Comment:
How about renaming variable name to `ErrFilterNotFound` as well?

Reply:
Resolved along with the merge 👍